### ISSUE TITLE
Evaluation Procedure

### ISSUE THREAD

**Issue by RudolfReiter** [shamilmamedov/flexible_arm#29]
We need a script or a workflow to take one of the configurations

- RL + environment
- MPC + enviornment
- safety filter  + environment

with options 

- random wall / fixed wall
- sampling close to wall / sampling randomly in feasible region

and collect the following data

- computation time of algorithm
- constraint violation of wall
- time to reach epsilon region
- closed loop cost as defined in MPC (Q,R)

**Comment by Erfi** [comment]
By computation time of algorithm are we referring to the "inference time" i.e. the time it takes the algorithms to give us the next action?

**Comment by Erfi** [comment]
Do we want to add random walls? If so I revise my comment on #31 and say that we should add the wall parameters also to the observation space.

**Comment by RudolfReiter** [comment]
Yes, inference time. I had problems with getting rid of all the pytorch specific things and really just count the time for matrix multiplications. Maybe you have a good idea for how to get the best estimate for embedded computation @Erfi ?

First, I would suggest to keep the wall constant. But have it already accordingly in the environment.
